client/components/PubEdge/PubEdge.js

```diff
@@ -109,41 +109,52 @@ const PubEdge = (props) => {
 		onKeyDown: handleClick,
 		role: 'link',
 		tabIndex: '0',
+		alt: title,
 	};
 
 	const maybeLink = (element, restProps = {}) => {
 		if (actsLikeLink) {
-			return element;
+			return (
+				<span {...restProps} className={classNames(restProps.className, 'link')}>
+					{element}
+				</span>
+			);
```

Comment:
`maybeLink` now returns a "false" link, which is just a span with the class `link` and whatever props the function was passed.